approval_polls/templates/approval_polls/create.html

```diff
@@ -59,6 +59,10 @@ <h4 class='panel-title'>
           <div class='radio'>
             <label><input type='radio' name='radio-poll-type' checked='true' value='2'>Only users with registered accounts (restricted to one ballot per user, requires login).</label>
           </div>
+          <div class = 'form-group'>
+            <p>Please select a closing date and time for this poll, if any</p>
+            <input id = 'datetimepicker' type="text" class="form-control" name = 'close-datetime' value= '' placeholder = 'Poll Closing Date'>
```

Comment:
Minor formatting issue here. Since we are using the form attribute='value' throughout the file,  for consistency, we should avoid mixing `"` and `'` , and spaces around the '='.